common/src/main/java/org/opensearch/ml/common/MLModel.java

```diff
@@ -243,6 +248,9 @@ public MLModel(StreamInput input) throws IOException {
                 connector = Connector.fromStream(input);
             }
             connectorId = input.readOptionalString();
+            if (input.readBoolean()) {
```

Comment:
check version for bwc, you can sync up with @b4sjoo

Reply:
Example PR could be: https://github.com/opensearch-project/ml-commons/pull/1944/files#diff-61645cbc8c2a190887ce3d5f2da602d7e862802dd483aa545bf64b1c5bfbbe2fR32

Reply:
@jngz-es @ylwu-amzn @dhrubo-os The version control need to do in any *Input.java file, but no need to do here.

Reply:
Thanks @b4sjoo for confirming.